fastlane/CommonFastfile

```diff
@@ -142,11 +146,43 @@ lane :github_changelog do |options|
       item = items.first
       message = "#{item['title']} (##{item['number']})"
       username = item["user"]["login"]
+      change_types = item["labels"].map do |label|
+        label_name = label["name"]
+        (supported_types.include? label_name) ? label_name : "other"
+      end.uniq
```

Comment:
I'm no the biggest expert on Ruby. To make sure I follow this, looks like it will only include PRs with the labels in `supported_types`, but categorize the ones that aren't in `changelog_sections` as "other"?